src/payment/strategies/klarna/klarna-payment-strategy.ts

```diff
@@ -121,6 +132,35 @@ export default class KlarnaPaymentStrategy implements PaymentStrategy {
             }));
     }
 
+    private _getUpdateSessionData(billingAddress: BillingAddress, shippingAddress?: Address): KlarnaUpdateSessionParams | {} {
```

Comment:
Same as before, if `KlarnaUpdateSessionParams` can be completely empty, the type should reflect that.